src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar">
+      <div class="day day--1 calendar__day day--sun"></div>
+      <div class="day day--2 calendar__day"></div>
+      <div class="day day--3 calendar__day"></div>
+      <div class="day day--4 calendar__day"></div>
+      <div class="day day--5 calendar__day"></div>
+      <div class="day day--6 calendar__day"></div>
+      <div class="day day--7 calendar__day"></div>
+      <div class="day day--8 calendar__day"></div>
+      <div class="day day--9 calendar__day"></div>
```

Comment:
Don't need add modifiers for days. Use nth-child
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```